packages/paypal-express-integration/src/paypal-express-integration-service.ts

```diff
@@ -0,0 +1,11 @@
+import PayPalExpressScriptLoader from './paypal-express-script-loader';
+
+export default class PayPalExpressIntegrationService {
```

Comment:
I don't think we need this file for current integration package

Reply:
removed